include/panel_state_manager.hpp

```diff
@@ -199,6 +207,13 @@ class PanelStateManager
      */
     void updateFunctionStatus();
 
+    /**
+     * @brief API which tells if the function is enabled or not
+     * @param[in] funcNum - Function number
+     * @return (1/0/-1) if function is enabled/disabled/invalid respectively.
```

Comment:
Previous implementation was ok. This function needs to just return if the given function number is enabled or not.

Reply:
done